docs/source/add_new_adaptor.md

```diff
@@ -0,0 +1,243 @@
+
+
+How to Support New Adaptor
+=======
+
+
+- [Introduction](#introduction)
+- [Add query_fw_capability to Adaptor](#add-query-fw-capability-to-adaptor)
+- [Add Pre-quantization Optimizations](#add-pre-quantization-optimizations)
+- [Quantize Model according to tune cfg](#quantize-model-according-to-tune-cfg)
+- [Evaluate Model on Validation Dataset](#evaluate-model-on-validation-dataset)
+- [Train Model for Quantization-aware Training](#Train-model-for-quantization-awar-training)
+- [Other API that not Mandatory Needed](#other-api-that-not-mandatory-needed)
+- [Summary](#summary)
+
+## Introduction
+Intel® Neural Compressor builds the low-precision inference solution on popular deep learning frameworks such as TensorFlow, PyTorch, MXNet, Keras and ONNX Runtime. The adaptor layer is the bridge between the tuning strategy and vanilla framework quantization APIs.
+
+The quantizable operator behavior and it's tuning scope is defined in specific framework YAML file. The adaptor will parse this file and give the quantization capability to the Strategy object. Then Strategy will build tuning space of the specific graph/model and generate different tuning configuration from the tuning space to adaptor.
+
+The diagram below illustrates all the relevant steps of how adaptor is invoked, with additional details provided for each annotated step.
```

Comment:
we should introduce what's the interface in adaptor layer at first, and then the working flow how they interactive each other